sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2147,6 +2147,12 @@ pi_result piDeviceRelease(pi_device Device) {
 pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
                           size_t ParamValueSize, void *ParamValue,
                           size_t *ParamValueSizeRet) {
+  // TODO: This is a work-around for missing UR info query. Replace with propper
+  //       UR mapping when available.
```

Comment:
Tagging @smaslov-intel, @jandres742, & @kbenzie - Changes are in draft, but feel free to start reviewing the PI interfaces.

Reply:
Given there are a lot of additions here I think it would be useful to present this to the UR WG, if you could @steffenlarsen?